### PR TITLE
Add ability to modify num_hidden_layers from benchmark script

### PR DESCRIPTION
**Pull request by wangyems**
**Description**: Describe your changes.

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: onnxruntime/python/tools/transformers/onnx_exporter.py [hunk 1]
```diff
@@ -266,6 +266,11 @@ def load_pt_model(model_name, model_class, cache_dir):
     if hasattr(config, 'return_dict'):
         config.return_dict = False
 
+    num_layers = os.getenv('NUM_HIDDEN_LAYERS')
+    if num_layers is not None:
+        config.num_hidden_layers = int(num_layers)
```
Comment (tianleiwu) [inline onnxruntime/python/tools/transformers/onnx_exporter.py:271]: Some model like bart has also other attributes like encoder_layers, decoder_layers. Did you try it on encoder decoder model?

**Review by tianleiwu — APPROVED** [review]

**Comment by tianleiwu** [comment]
It works in this way using environment variable, but it might be better to add a parameter in command line like other parameters (batch_size, sequence_length etc). For example, change could be like

Update benchmark.py
- add argument "--force_layers", default value is None
- pass through the parameter to exporter:
      bencharmk.run_pytorch(..., force_layers)
      onnx_exporter.export_onnx_model_from_pt(..., force_layers)
- output the parameter to JSON (https://github.com/microsoft/onnxruntime/blob/4d6d4dfb9de610fbb09e23b36379473b4005a99d/onnxruntime/python/tools/transformers/benchmark.py#L248) and result TSV

May need check force_layers <= layers in model configuration.
Also update encoder_layers, decoder_layers if the attributes exist in model configuration.

File: onnxruntime/python/tools/transformers/onnx_exporter.py (hunk 1 above)
Comment (wangyems) [reply onnxruntime/python/tools/transformers/onnx_exporter.py]: Added encoder_layers and decoder_layers

**Comment by wangyems** [comment]
> It works in this way using environment variable, but it might be better to add a parameter in command line like other parameters (batch_size, sequence_length etc). For example, change could be like
> 
> Update benchmark.py
> 
> * add argument "--force_layers", default value is None
> * pass through the parameter to exporter:
>   run_onnxruntime(..., force_layers)
>   run_pytorch(..., force_layers)
>   onnx_exporter.export_onnx_model_from_pt(..., force_layers)
> * output the parameter to JSON (https://github.com/microsoft/onnxruntime/blob/4d6d4dfb9de610fbb09e23b36379473b4005a99d/onnxruntime/python/tools/transformers/benchmark.py#L248
>   )
> * add a column force_layers in the result TSV
> 
> May need check force_layers <= layers in model configuration. Also update encoder_layers, decoder_layers if the attributes exist in model configuration.

parameterize it. 
it's ok if force_layers > layers from benchmarking perspective

**Review by tianleiwu — APPROVED** [review]

**Pull request merged**